doc/source/ray-air/examples/batch_tuning.ipynb

```diff
@@ -0,0 +1,1197 @@
+       "            </g>\n",
+       "        </svg>\n",
+       "        <table>\n",
+       "            <tr>\n",
+       "                <td style=\"text-align: left\"><b>Python version:</b></td>\n",
+       "                <td style=\"text-align: left\"><b>3.8.13</b></td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "                <td style=\"text-align: left\"><b>Ray version:</b></td>\n",
+       "                <td style=\"text-align: left\"><b> 2.0.1</b></td>\n",
+       "            </tr>\n",
+       "            <tr>\n",
+       "    <td style=\"text-align: left\"><b>Dashboard:</b></td>\n",
+       "    <td style=\"text-align: left\"><b><a href=\"http://127.0.0.1:8265\" target=\"_blank\">http://127.0.0.1:8265</a></b></td>\n",
+       "</tr>\n",
+       "\n",
+       "        </table>\n",
+       "    </div>\n",
+       "</div>\n"
+      ],
+      "text/plain": [
+       "RayContext(dashboard_url='127.0.0.1:8265', python_version='3.8.13', ray_version='2.0.1', ray_commit='03b6bc7b5a305877501110ec04710a9c57011479', address_info={'node_ip_address': '127.0.0.1', 'raylet_ip_address': '127.0.0.1', 'redis_address': None, 'object_store_address': '/tmp/ray/session_2022-11-09_11-17-34_825866_27490/sockets/plasma_store', 'raylet_socket_name': '/tmp/ray/session_2022-11-09_11-17-34_825866_27490/sockets/raylet', 'webui_url': '127.0.0.1:8265', 'session_dir': '/tmp/ray/session_2022-11-09_11-17-34_825866_27490', 'metrics_export_port': 64093, 'gcs_address': '127.0.0.1:62719', 'address': '127.0.0.1:62719', 'dashboard_agent_listen_port': 52365, 'node_id': 'f047796bbbc91fada43c063b5df319a587610c12949637c3ea701b81'})"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import ray\n",
+    "\n",
+    "if ray.is_initialized():\n",
+    "    ray.shutdown()\n",
+    "ray.init()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "d54a9528-defe-4102-a179-d779f0dc2f66",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'CPU': 8.0, 'node:127.0.0.1': 1.0, 'object_store_memory': 2147483648.0, 'memory': 8644909466.0}\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(ray.cluster_resources())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9c86a93c-cea6-4dbc-aebc-9a1623734d3d",
+   "metadata": {},
+   "source": [
+    "<b>Scikit-Learn</b> is one of the most widely used tools in the ML community for working with data, offering dozens of easy-to-use machine learning algorithms. \n",
+    "> However, to achieve high performance, you often need to perform model selection and checkpointing. \n",
+    "\n",
+    "[Train-sklearn libraries](https://docs.ray.io/en/latest/tune/examples/tune-sklearn.html) make these tasks more convenient in a distributed setting.  Below, we import sklearn, ray Tune and AIR, and our custom Train-sklearn libraries.\n"
```

Comment:
```suggestion
    "<b>scikit-learn</b> is one of the most widely used tools in the ML community for working with data, offering dozens of easy-to-use machine learning algorithms. \n",
    "[Ray AIR](https://docs.ray.io/en/latest/tune/examples/tune-sklearn.html) makes training, tuning and inference with those algorithms more convenient in a distributed setting.  Below, we import sklearn and Ray AIR, including framework-specific objects such as `SklearnCheckpoint`.\n"
```

Reply:
Due to recent code nightly pushes, we were able to get rid of `SklearnCheckpoint` and keep the notebook using pure AIR Checkpoints instead.